src/main/java/com/adyen/service/LegalEntityManagement/BusinessLineService.java

```diff
@@ -0,0 +1,39 @@
+package com.adyen.service.LegalEntityManagement;
+
+import com.adyen.Client;
+import com.adyen.Service;
+import com.adyen.constants.ApiConstants;
+import com.adyen.model.legalEntityManagement.BusinessLine;
+import com.adyen.model.legalEntityManagement.BusinessLineInfo;
+import com.adyen.model.legalEntityManagement.JSON;
+import com.adyen.service.exception.ApiException;
+import com.adyen.service.resource.LegalEntityManagementResource;
+
+import java.io.IOException;
+
+public class BusinessLineService extends Service {
+    public BusinessLineService(Client client) {
+        super(client);
+        new JSON();
```

Comment:
Let's instantiate `JSON` only once in `com.adyen.service.LegalEntityManagementService`.

Reply:
Smart!